filebeat/beat/filebeat.go

```diff
@@ -101,17 +112,18 @@ func (fb *Filebeat) Cleanup(b *beat.Beat) error {
 // Stop is called on exit for cleanup
 func (fb *Filebeat) Stop() {
 
-	// Stop harvesters
-	// Stop prospectors
+	logp.Info("Stopping filebeat")
+	// Stop crawler -> stop prospectors -> stop harvesters
+	fb.cralwer.Stop()
```

Comment:
Good point. I just tried it out locally and unfortunately it exactly happens what you described. Any suggestion on how we should solve it?